sale_automatic_workflow/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# © 2011 Akretion Sébastien BEAU <sebastien.beau@akretion.com>
+# © 2013 Camptocamp SA (author: Guewen Baconnier)
+# © 2016 Sodexis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Sale Automatic Workflow',
+    'version': '11.0.1.0.0',
+    'category': 'Sales Management',
+    'license': 'AGPL-3',
+    'author': "Akretion,Camptocamp,Sodexis,Odoo Community Association (OCA)",
+    'website': 'http://www.akretion.com/',
+    'depends': ['sale_stock', 'sales_team',
+                ],
+    'data': ['views/sale_view.xml',
```

Comment:
aaand this one too

Reply:
what do you mean exactly? The formatting is fine and passing flake8

Reply:
I was talking about putting each dependency on its own line, that makes changes clearer if a list of dependencies changes for whatever reason
For example,
```python
'depends': [
    'sale_stock',
    'sales_team',
],
```
and the `data` list may change even more often
This is fine enough though :)